cmd/lnmuxd/run.go

```diff
@@ -106,9 +109,10 @@ func runAction(c *cli.Context) error {
 		return err
 	}
 
-	// Instantiate grpc server.
+	// Instantiate grpc server and enable reflection and Prometheus metrics.
 	grpcServer := grpc.NewServer()
 	reflection.Register(grpcServer)
+	grpc_prometheus.Register(grpcServer)
```

Comment:
Is this all that's needed? I thought there were also interceptors that need to be registered? I also don't think I see the calls to lnmux in the metrics.

For outgoing calls to lnd it may already work. For that, interceptors were added already. (`loadGrpcConn`)

Reply:
Sorry, I should have made this into a draft - I'm still working on this.

Reply:
I've added interceptors when the server is created, and checked that both the outgoing calls to lnd and incoming calls from an lnmux client are counted in metrics.